### ISSUE TITLE
Prismatik not working with driver update wch.cn - Ports - 3.8.2023.2

### ISSUE THREAD

**Issue by terrainer** [psieg/Lightpack#586]
Just a heads up for everyone.

Windows 11 updated my COM3 driver to wch.cn - Ports - 3.8.2023.2, causing Prismatik to be unable to control the LED strip I was using.

Everything is working again after going to Device Manager > Ports (COM & LPT) > USB-SERIAL CH340 properties > Driver > Roll Back Driver

Hope this helps anyone who is having trouble after recent updates.

**Comment by Muzzygizmo** [comment]
I agree, when downgrading the driver to 3.5.2019.1 everything is back to normal.
https://www.catalog.update.microsoft.com/Search.aspx?q=CH340
Thank you

**Comment by ewanotoole** [comment]
> I agree, when downgrading the driver to 3.5.2019.1 everything is back to normal. https://www.catalog.update.microsoft.com/Search.aspx?q=CH340 Thank you

Thank you SO much for finding this.
